Userland/Libraries/LibGfx/ImageFormats/JPEGXLEntropyDecoder.h

```diff
@@ -55,11 +55,7 @@ class EntropyDecoder {
 
 public:
     EntropyDecoder() = default;
-    ~EntropyDecoder()
-    {
-        if (m_state.has_value() && *m_state != 0x130000)
-            dbgln("JPEGXLLoader: ANS decoder left in invalid state");
-    }
+    ~EntropyDecoder() = default;
```

Comment:
should the dtor check that `ensure_end_state` has been called before it runs?

(possible to forget to call it, otherwise)

Reply:
That's what I had but it's a bit too noisy. Because if there are any error that bubbles up during a read (like inside `read_modular_bitstream()`) the check won't pass.